packages/agoric-cli/tools/getting-started.js

```diff
@@ -23,6 +24,38 @@ const dirname = new URL('./', import.meta.url).pathname;
 // yarn start:contract
 // yarn start:ui
 
+/**
+ * @param {string} url
+ * @returns {Promise<bigint>}
+ */
+const getLatestBlockHeight = url =>
```

Comment:
I wonder how many implementations we have of this by now :)

Reply:
It occured to me: I should probably do a pure JS implementation available via the `agoric wait-for-block` command  to subsume the functionality of the `waitForBlock` shell function and the various polling loops we have in Makefiles, etc.